tensorflow/python/eager/def_function.py

```diff
@@ -99,7 +99,7 @@ def __call__(self, x):
 FREQUENT_TRACING_WARNING_MAX_CALL_HISTORY = 10
 FREQUENT_TRACING_WARNING_THRESHOLD = 5
 FREQUENT_TRACING_WARNING_MAX_WARNING_PER_DETECTOR = 2
-
+PR_49310 = False
```

Comment:
Naming nit: let's be more specific with this one. Can add the PR/issue number as a comment. How about: `ALLOW_DYNAMIC_VARIABLE_CREATION = False`

Reply:
Yes I supposed that PR_49310 was for you short term test. Now is `ALLOW_DYNAMIC_VARIABLE_CREATION`